web-client/src/presenter/actions/getJudgesChambersAction.ts

```diff
@@ -0,0 +1,21 @@
+import { JudgeChambersInfo } from '@shared/proxies/users/getJudgesChambersProxy';
+import { isEmpty, sortBy } from 'lodash';
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const getJudgesChambersAction = async ({
+  applicationContext,
+  get,
+}: ActionProps) => {
+  const judgesChambersCached = get(state.judgesChambers);
```

Comment:
Updates to chambers are relatively infrequent, so I think it is best to cache. Happy to hear other opinions.

Reply:
Agree